src/components/date-picker-month-header/date-picker-month-header.scss

```diff
@@ -117,7 +117,8 @@
 }
 
 .year--suffix {
-  @apply w-16 text-left;
+  @apply w-16
+  text-start;
```

Comment:
According to https://github.com/Esri/calcite-components/pull/5344, this needs to stay the same. cc @anveshmekala

Reply:
Thank you! I missed that.